src/main.js

```diff
@@ -318,6 +318,7 @@ function getArgvConfigPath() {
 		dataFolderName = `${dataFolderName}-dev`;
 	}
 
+	// @ts-ignore
```

Comment:
Was a new error introduced here?

Reply:
I think the error was there before. The better TypeScript got with checking JS, the more errors came up in these JS files, but they never end up causing the build to fail so it is a catchup game.

Reply:
Yeah, the error was there before. That's was the last one in that file but all files have some kind of validation/correctness issue. TBH I think we would be better off by implementing all of this in TS and have a dedicated CU that emits the JS code for this.